src/main/res/values/strings.xml

```diff
@@ -150,4 +151,11 @@
     <string name="intro_title_6">Ready</string>
     <string name="intro_desc_6">You\'re all set to go. Thanks for using Amahi.</string>
 
+    <!--Dialog File Info-->
+    <string name="dialog_file_name">Name:</string>
+    <string name="dialog_file_type">File Type:</string>
+    <string name="dialog_file_modification">Last Modification:</string>
```

Comment:
I guess it should be "Last Modified" or just "Modified".

Reply:
oops! My bad.